lib/usd/translators/CMakeLists.txt

```diff
@@ -17,14 +17,12 @@ target_sources(${TARGET_NAME}
     PRIVATE
         cameraReader.cpp
         cameraWriter.cpp
-        fileTextureWriter.cpp
         instancerWriter.cpp
         jointWriter.cpp
         lightReader.cpp
         lightWriter.cpp
         locatorWriter.cpp
         materialReader.cpp
-        materialWriter.cpp
```

Comment:
As requested in a previous review: split the monolithic materialWriter.cpp file per dependency node.